c7n/resources/ses.py

```diff
@@ -88,3 +91,51 @@ class resource_type(TypeInfo):
         permission_prefix = 'ses'
         arn_service = 'ses'
         cfn_type = 'AWS::SES::EmailIdentity'
+
+
+@SESEmailIdentity.filter_registry.register('has-statement')
+class HasStatementFilter(polstmt_filter.HasStatementFilter):
+
+    def get_std_format_args(self, email_identity):
+        return {
+            'account_id': self.manager.config.account_id,
+            'region': self.manager.config.region,
+            'email_identity_name': email_identity['IdentityName'],
+        }
+
+    def process_resource(self, email_identity):
+        policies_attribute = getattr(self, 'policy_attribute', 'Policies')
```

Comment:
this should be more explicit, self.policy_attribute, and set the class variable if its trying to be an override from the base class.